src/Host/Broker/Impl/About/LoadController.cs

```diff
@@ -37,62 +32,30 @@ public class LoadController : Controller {
             return _hostLoad;
         }
 
-        private void UpdateMeasurement() {
-            var query = new SelectQuery(@"Select * from Win32_OperatingSystem");
-            using (var search = new ManagementObjectSearcher(query)) {
-                foreach (var mo in search.Get()) {
-                    var totalMemory = GetSizeInGB(mo, "TotalVisibleMemorySize");
-                    var freeMemory = GetSizeInGB(mo, "FreePhysicalMemory");
-                    _hostLoad.MemoryLoad = (double)(totalMemory - freeMemory) / (double)totalMemory;
-                    break;
-                }
-            }
-
-            _hostLoad.CpuLoad = GetCpuLoad();
+        private void UpdateMeasurement(object state) {
+            var memoryInfo = _systemInfo.GetMemoryInformation();
+            _hostLoad.MemoryLoad = (double)(memoryInfo.TotalPhysicalMemory - memoryInfo.FreePhysicalMemory) / memoryInfo.TotalPhysicalMemory;
+            _hostLoad.CpuLoad = _systemInfo.GetCpuLoad();
             _hostLoad.NetworkLoad = GetNetworkLoad();
         }
 
         protected override void Dispose(bool disposing) {
-            _timer?.Stop();
             _timer?.Dispose();
             base.Dispose(disposing);
         }
 
-        private void OnTimer(object sender, ElapsedEventArgs e) {
-            UpdateMeasurement();
-        }
-
-        private long GetSizeInGB(ManagementBaseObject mo, string name) {
-            int result;
-            var x = mo[name].ToString();
-            return Int32.TryParse(x, out result) ? result / 1024 : 0;
-        }
-
-        private double GetCpuLoad() {
-            double counter = 0;
-            int iterations = 5;
-
-            for (int i = 0; i < iterations; i++) {
-                Thread.Sleep(200);
-                counter += _cpuCounter.NextValue();
-            }
-
-            return counter / (iterations * 100);
-        }
-
         private double GetNetworkLoad() {
             if (!NetworkInterface.GetIsNetworkAvailable()) {
                 return 0;
             }
 
             // Select compatible active network interfaces
             var interfaces = NetworkInterface.GetAllNetworkInterfaces()
-                                             .Where(x => x.Speed > 0 &&
-                                                         x.Supports(NetworkInterfaceComponent.IPv4) &&
-                                                         x.Supports(NetworkInterfaceComponent.IPv6) &&
-                                                         x.OperationalStatus == OperationalStatus.Up &&
-                                                         x.Speed > 0 &&
-                                                         IsCompatibleInterface(x.NetworkInterfaceType)).ToArray();
+                .Where(x => x.Speed > 0 
```

Comment:
This throws Platform Not supported exception on linux. Sadly there is no way to provide this information in a generic way.